README.md

```diff
@@ -21,17 +24,18 @@ New features like a new network concept will be part of future releases.
 * the LAN-interface (br-lan) is not providing a default route on a unconfigured node, so manual reconfiguration is required when not using the assistent
 * the assistent will not offer the setup of a private AP anymore
 * when changing or initially confiuring VLANs (via LuCI) the interfacenames will also be changed (should fix #388)
-* a lot of security-fixes for linux and many packages (since Kathleen 0.3.0)
 
 ## Features
-* based on [OpenWrt](https://openwrt.org/start) v17.01.4+ (lede-17.01 branch)
-  * Linux 4.4.116
+* based on [OpenWrt](https://openwrt.org/start) v17.01.6+ (lede-17.01 branch)
+  * Linux 4.4.167
   * OLSR 0.9.0.3 (downgraded for BBB-VPN compatibility)
-  * B.A.T.M.A.N. 2016.5
+  * B.A.T.M.A.N. 2016.5 (with patches of 2018.4)
 * custom package lists for different settings
   * "default" variant includes ffwizard, openvpn, BATMAN
```

Comment:
Perhaps it would make sense to call "default" the no-tunnel version.  this is unclear from the current description.

Additionally, almost all the other image types have "as default uplink" in the description.  It might make sense to add this to the "default" image too.

Reply:
https://github.com/freifunk-berlin/firmware/issues/534